op-conductor/conductor/service.go

```diff
@@ -647,9 +647,13 @@ func (oc *OpConductor) action() {
 
 	oc.log.Debug("exiting action with status and error", "status", status, "err", err)
 	if err != nil {
-		oc.log.Error("failed to execute step, queueing another one to retry", "err", err, "status", status)
-		time.Sleep(oc.retryBackoff())
-		oc.queueAction()
+		select {
+		case <-oc.shutdownCtx.Done():
+		case <-time.After(oc.retryBackoff()):
```

Comment:
Wouldn't you now wait two times for a `oc.retryBackoff()`? You call a `time.Sleep` when entering this block. So maybe we want to do this instead?
```suggestion
		default:
```

Reply:
Ah, no I've just forgotten to remove the actual time.Sleep. Thanks